awx/main/models/unified_jobs.py

```diff
@@ -1512,3 +1512,35 @@ def log_lifecycle(self, state, blocked_by=None):
         else:
             msg = f"{self._meta.model_name}-{self.id} {state.replace('_', ' ')}"
         logger_job_lifecycle.debug(msg, extra=extra)
+
+    @property
+    def launched_by(self):
+        ancestor_job = self.ancestor_job
+        if ancestor_job.launch_type == "manual" or ancestor_job.launch_type == "relaunch":
+            return ancestor_job.created_by
+        elif ancestor_job.launch_type == "scheduled":
+            return ancestor_job.schedule
+        elif ancestor_job.launch_type == "workflow":
+            return ancestor_job.get_workflow_job()
+        elif ancestor_job.launch_type == "dependency":
+            if ancestor_job.job_type_name == "project_update":
+                return ancestor_job.project
```

Comment:
@jbradberry not sure I fully understand why its getting set to the project, when really it is being launched as a dependency of the job template launch.

Reply:
Not sure why I'm getting @ mentioned, this isn't my PR.  I offered some advice, but if other people know of a better model relationship to be used, by all means talk it out.

Reply:
https://github.com/ansible/awx/blob/f850f8d/awx/main/scheduler/task_manager.py#L352-L381

By my reading of this code, we don't store any information about what job triggered the update for either case.

Reply:
k, we will stick with this then